packages/insomnia-importers/bin/cli

```diff
@@ -1,3 +1,2 @@
-#!/usr/bin/env node
-
-require('../src/cli').go();
+#!/bin/sh
+chmod +x /home/dimitri/src/github.com/Kong/insomnia/packages/insomnia-importers/dist/src/cli && exec /home/dimitri/src/github.com/Kong/insomnia/packages/insomnia-importers/dist/src/cli "$@"
```

Comment:
This file references your home directory structure

Reply:
I have no idea whatsoever how this happened (i.e. what tool did this).  I know I saw this file many times while testing and I never saw it like this.. all the same..  I'll definitely fix.

Reply:
resolved in https://github.com/dimitropoulos/insomnia/pull/4/commits/552d5ec20193f79b013914cc3c542c8d686c7fdd